src/main/g8/src/main/scala/com/lightbend/akka/http/sample/JsonSupport.scala

```diff
@@ -0,0 +1,9 @@
+package com.lightbend.akka.http.sample
+
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+import spray.json.DefaultJsonProtocol
+
+trait JsonSupport extends SprayJsonSupport with DefaultJsonProtocol {
```

Comment:
We might keep it a trait that extends from `SprayJsonSupport` but extending from `DefaultJsonProtocol` is an antipattern we need to avoid. Otherwise, it's very easy to run into implicit resolution problems.

The recommendation is usually to write it like this and import it where needed.

```
object JsonSupport {
  import DefaultJsonProtocol._
  implicit val userJsonFormat = jsonFormat3(User)
  implicit val usersJsonFormat = jsonFormat1(Users)
}
```